packages/gasket-data/README.md

```diff
@@ -37,12 +37,13 @@ object's `locals.gasketData` property.
 For example, when using the [middleware lifecycle] in a plugin:
 
 ```js
-module.exports = {
+export default {
```

Comment:
I think his entire **Adding Data** section should be removed as the pattern has changed significantly from setting res.locals. Just link to the [publicGasketData lifecycle](https://github.com/godaddy/gasket/tree/v7/packages/gasket-plugin-data#publicgasketdata) for updating what public data should be available in the browser.